server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -197,10 +211,50 @@ function markAppointmentAsReady($token, $firstName, $lastName, $emailAddress, $p
 	echo json_encode($response);
 }
 
+function storeConsent($reviewConsent, $virtualConsent, $signature, $appointmentId) {
+	GLOBAL $DB_CONN;
+
+	$response = array();
+	$response['success'] = false;
+	
+	if($virtualConsent === 'true' && $virtualConsent !== null && $signature !== null && $signature !== '' && $appointmentId !== null) {
+		$reviewConsent = $reviewConsent === 'true' ? 1 : 0;
+		$virtualConsent = $virtualConsent === 'true' ? 1 : 0;
+		try {
+			$DB_CONN->beginTransaction();
+
+			insertConsent($reviewConsent, $virtualConsent, $signature, $appointmentId);
+			$DB_CONN->commit();
+
+			$response['success'] = true;
+		} catch (Exception $e) {
+			$DB_CONN->rollback();
+			$response['message'] = 'An error occurred while trying to record your consent. Please try again in a few minutes.';
+		}
+	} else {
+		$response['message'] = 'Your consent information was not found to be valid. Please try again in a few minutes.';
```

Comment:
Maybe update this message to not say "try again in a few"? Maybe just say it was invalid, double check the information and submit again

Reply:
That makes sense, I did something similar in the controller